README.md

```diff
@@ -120,6 +120,32 @@ make install
 
 Please see our [additional configuration instructions](http://docs.timescale.com/getting-started/installation#update-postgresql-conf).
 
+#### Building from source (Windows)
+
+**Prerequisites**:
+
+- A standard [PostgreSQL 9.6 64-bit installation](https://www.enterprisedb.com/downloads/postgres-postgresql-downloads#windows) 
+- Microsoft Visual Studio
+- [CMake](https://cmake.org/) version 3.4 or greater
```

Comment:
CMake is part of Visual Studio 2017 AFAIK. Should we just require that VS version and remove the CMake dependency? Or, write VS 2017 or CMake + earlier CMake?